src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -431,6 +438,46 @@ def get_extra_inputs_as_dict(args: argparse.Namespace) -> dict:
     return request_inputs
 
 
+def _parse_compare_args(subparsers) -> argparse.ArgumentParser:
+    compare = subparsers.add_parser(
+        "compare", help="Generate plots that compare multiple runs."
```

Comment:
```suggestion
        "compare", help="Generate plots that compare multiple profile runs."
```

Reply:
Updated.

Reply:
I think some of your changes didn't get pushed, including this one.